### PR TITLE
Refactoring js

### PR DESCRIPTION
**Pull request by vsn4ik**

### CODE REVIEW

File: js/bootstrap-datepicker.js [hunk 1]
```diff
@@ -141,13 +141,12 @@
...
 
-		this.viewMode = this.o.startView;
-
-		if (this.o.calendarWeeks)
-			this.picker.find('thead .datepicker-title, tfoot .today, tfoot .clear')
-						.attr('colspan', function(i, val){
-							return parseInt(val) + 1;
-						});
+		if (this.o.calendarWeeks) {
+			this.picker.find('.datepicker-days .datepicker-switch, thead .datepicker-title, tfoot .today, tfoot .clear')
```
Comment (acrobat) [inline js/bootstrap-datepicker.js]: Why are the extra classes needed?

Comment (vsn4ik) [inline js/bootstrap-datepicker.js]: `.datepicker-days`?

Comment (vsn4ik) [inline js/bootstrap-datepicker.js]: Adding columns occurs only in `.datepicker-days`.

Comment (acrobat) [inline js/bootstrap-datepicker.js]: Yes the `.datepicker-days .datepicker-switch` classes. Don't think this is wrong, but just curious why they are added!

Comment (vsn4ik) [inline js/bootstrap-datepicker.js]: I moved this. See below: https://github.com/eternicode/bootstrap-datepicker/pull/1893/commits/4c70acaad66f3bdbbdd91d0c70b44d73c535c223#diff-dd513a8bab7ad1033c8784c4a1b9ce15L828

Comment (acrobat) [inline js/bootstrap-datepicker.js]: ah yes i see!

**Comment by vsn4ik** [comment]
I hope not to have to wait another month...

**Comment by vsn4ik** [comment]
Thanks!

**Pull request merged**